src/game.py

```diff
@@ -94,5 +94,6 @@ def play_game(self):
 
     def run(self):
         message.display_welcome_message(self)
+        message.print_rules(self)
```

Comment:
building off my other comment, this line would probably look something more like this ...
```suggestion
        ui.display_rules(self)
```